src/DimensionalData.jl

```diff
@@ -24,6 +24,7 @@ import Adapt,
        ArrayInterface,
        ConstructionBase, 
        Extents,
+       InvertedIndices,
        IteratorInterfaceExtensions,
        RecipesBase,
        PrecompileTools,
```

Comment:
You need to export also `Not` in line 52. Otherwise is not available at the DD level.